src/utilities/transition/Transition.constants.ts

```diff
@@ -0,0 +1,65 @@
+import { css } from 'styled-components';
+
+import { TThemeAnimationSpeed } from '../theme';
+
+import { TTransitionType, TTransitionTypeStatusMap } from './Transition.types';
+
+const TRANSITION_SPEEDS: TThemeAnimationSpeed[] = [
+    'instant',
+    'fastest',
+    'fast',
+    'normal',
+    'slow',
+    'slowest',
+];
+
+const DEFAULT_TRANSITION_SPEED: TThemeAnimationSpeed = 'normal';
+
+const TRANSITION_TYPE_PROPERTY_MAP: Record<TTransitionType, string> = {
```

Comment:
this is really nice to have. I assume we can add more values as needed so we can include all of our transitions here?

Reply:
Definitely 👍🏻